xlators/mgmt/glusterd/src/glusterd-op-sm.c

```diff
@@ -2514,6 +2515,35 @@ glusterd_set_brick_mx_opts(dict_t *dict, char *key, char *value,
     return ret;
 }
 
+static int
+glusterd_set_brick_graceful_cleanup(dict_t *dict, char *key, char *value,
+                                    char **op_errstr)
+{
+    int32_t ret = -1;
+    xlator_t *this = NULL;
+    glusterd_conf_t *priv = NULL;
+
+    this = THIS;
```

Comment:
You could pass `this` or even `this->private` or `this->private->opts` as an argument instead of using `THIS`.

Reply:
:+1: